src/app/investor/invest/invest.component.html

```diff
@@ -3,83 +3,117 @@
-                </div>
-            </div>
-
-          <div class="row">
-            <div class="col-xs-4 col-sm-4 col-md-4">
-                <p class="sw-box-text sw-confirmation-col-left">Company's wallet address:</p>
-                <p class="sw-box-text sw-confirmation-col-left">Investment:</p>
-                <p class="sw-box-text sw-confirmation-col-left">Gross return rate:</p>
-                <p class="sw-box-text sw-confirmation-col-left">Contract term:</p>
-                <p class="sw-box-text sw-confirmation-col-left">Assets:</p>
-
             </div>
-            <div class="col-xs-8 col-sm-8 col-md-8">
-                <p class="sw-box-text">{{investment.walletAddress}}</p>
-                <p class="sw-box-text">{{investment.totalAmount | currency}}</p>
-                <p class="sw-box-text">{{investment.grossReturn | percent}}</p>
-                <p class="sw-box-text">{{investment.paybackMonths}} months</p>
-                <p class="sw-box-text">{{investment.assets.length}}</p>
 
+            <div class="row sw-investment-info">
+              <div class="col-xs-8 col-sm-8 col-md-8 col-xs-offset-1 col-sm-offset-1 col-md-offset-1">
+                  <p *ngFor="let asset of investment.assets; let i = index" class="sw-box-text">
+                    Asset {{i + 1}}: <span class="sw-blue1">{{asset.value | currency:'USD':false:'.0-2'}} - {{collateralToString(i)}}</span>
+                  </p>
+                  <p class="sw-box-text">Gross return rate: 
+                    <span class="sw-blue1">{{investment.grossReturn | percent}}</span>
+                  </p>
+                  <p class="sw-box-text">Contract term: 
+                    <span class="sw-blue1">{{investment.paybackMonths}} months</span>
+                  </p>
+              </div>
             </div>
-          </div>
 
             <div class="row">
-              <div class=" col-sm-4 col-md-4">
+              <div class=" col-sm-5 col-md-5">
               </div>
-              <div class="col-sm-4 col-md-4">
+              <div class="col-sm-3 col-md-3">
                 <a class="sw-button sw-btn-ghost2" routerLink="/investor/offer/{{offerIndex}}">Return</a>
               </div>
-              <div class="col-sm-4 col-md-4">
-                <a class="sw-button sw-btn-ghost1" (click) = "confirmInvestment()">Confirm</a>
+              <div class="col-sm-3 col-md-3">
+                <a class="sw-button sw-btn-ghost1" (click) = "confirmInvestment()">Invest</a>
               </div>
             </div>
-
-           </div>
           </div>
 
         </div>
         </div> <!-- END -->
 
     </div>
+      <div class="col-sm-3 col-md-3 sw-company-box">
+        <div class="row">
+          <div class="col-sm-12 col-md-12">
+            <span>Company's wallet address: <p class="sw-blue1">{{investment.walletAddress}}</p></span>
+          </div>
+        </div>
+      <!-- 
+        <div class="row">
```

Comment:
remove coments

Reply:
I'm aware of this. There are loads of comments that should be removed.